src/pages/dmt5/_dmt5-video.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import { localize } from 'components/localization'
+import { Header, Image } from 'components/elements'
+import device from 'themes/device.js'
+
+const Container = styled.section`
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-top: 4rem;
+    align-items: center;
+
+    @media ${device.tabletL} {
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+`
+const TabsWrapper = styled.div`
+    width: 40.7rem;
+    margin-right: 2.6rem;
+`
+const VideoWrapper = styled.div`
+    max-width: 67.5rem;
+    width: 100%;
+`
+const Tab = styled.div`
+    width: 100%;
+    margin-top: 3rem;
+`
+const StepCommon = css`
+    font-weight: bold;
+    cursor: pointer;
+    padding-left: 1.8rem;
+    @media ${device.tabletL} {
+        text-align: left;
+        border: none;
+        margin-top: ${props => (props.no_margin ? '0' : '2rem')};
+    }
+`
+const Step1 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-1.png'
+            ? 'color: var(--color-black-3); border-left: 5px solid var(--color-red)'
+            : 'opacity: 0.2; border-left: 5px solid rgb(0, 0, 0, 0)'};
+`
+const Step2 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-2.png'
+            ? 'color: var(--color-black-3); border-left: 5px solid var(--color-red)'
+            : 'opacity: 0.2; border-left: 5px solid rgb(0, 0, 0, 0)'};
+`
+const Step3 = styled(Header)`
+    ${StepCommon}
+    ${props =>
+        props.current_step_image === 'dmt-5-step-3.png'
+            ? 'color: var(--color-black-3); border-left: 5px solid var(--color-red)'
+            : 'opacity: 0.2; border-left: 5px solid rgb(0, 0, 0, 0)'};
+`
```

Comment:
Why do we have 3 Step components here, when all are functioning the same?

Reply:
as I mentioned here https://github.com/binary-com/deriv-com/pull/365#discussion_r353593956 we can't use one component here